preseed.cfg

```diff
@@ -35,5 +39,4 @@ d-i network-console/password-again password kosagi
 d-i base-installer/kernel/skip-install boolean true
 nobootloader nobootloader/confirmation_common note
 
-# Finally configure where to boot from now on:
-d-i preseed/late_command string cp -r /lib/modules/3.19* /target/lib/modules/ ; cd /target ; tar xzf /hooks.tgz ; mkdir /sd ; mount /dev/mmcblk0p1 /sd ; mv /sd/uInitrd /sd/uInitrd-install ; umount /sd ; in-target update-initramfs -u -k all ; in-target /etc/kernel/postinst.d/zzz-novena-mkimage ; mount /dev/mmcblk0p1 /sd ; mkimage -A arm -T ramdisk -C none -n uInitrd -d /target/boot/initrd.img /sd/uInitrd ; cp /target/boot/initrd.img /sd/initrd.img ; sync ; umount /sd ; sync
```

Comment:
That would be a mistake on my part. Honestly, we can probably delete xorg.conf and get rid of that chuck in preseed.cfg (my Novena works just fine without xorg.conf).